Web/src/features/authentication/components/AuthErrorBoundary.tsx

```
@@ -0,0 +1,583 @@
+  };
+
+  /**
+   * Handle sign out action
+   */
+  private handleSignOut = async () => {
+    try {
+      // Import Clerk signOut dynamically to avoid circular dependencies
+      const { useAuth } = await import('@clerk/nextjs');
+      const { signOut } = useAuth();
+
+      await signOut();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)